.circleci/config.yml

```diff
@@ -28,8 +28,9 @@ workflows:
           base-revision: develop
           config-path: .circleci/main-config.yml
           mapping: |
-            pepper-apis/.* action "build-test"
+            .* action "build-test"
```

Comment:
To get a successful status check a workflow needs to run. This change ensures that we will trigger build-test workflow if ANY change in the repo has occurred.
So a change in a, e.g., study configuration file will trigger build-test even though we really don't have tests for these kind of files.